### PR TITLE
[RS][DDCE-1035] Linked up change buttons for 1st auth official's summ…

### PR DESCRIPTION
**Pull request by RafSobol**
…ary.

# DDCE-(StoryNumberHere)

##Bug fix / New feature (delete as appropriate)

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???

## Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

## Checklist PR Reviewer
 - [ ]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  I've squashed my commits - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

**Comment by RafSobol** [comment]
https://github.com/hmrc/charities-registration-journey-tests/pull/132

File: conf/address.routes [hunk 1]
```diff
@@ -6,8 +6,12 @@ GET        /charity-information-callback                 controllers.addressLook
 GET        /charity-information-postal-lookup            controllers.addressLookup.CharityPostalAddressLookupController.initializeJourney
 GET        /charity-information-postal-callback          controllers.addressLookup.CharityPostalAddressLookupController.callback(id: Option[String] ?= None)
 
-GET        /:index/authorised-official-lookup            controllers.addressLookup.AuthorisedOfficialsAddressLookupController.initializeJourney(index: Index)
-GET        /:index/authorised-official-callback          controllers.addressLookup.AuthorisedOfficialsAddressLookupController.callback(index: Index, id: Option[String] ?= None)
+GET        /:index/authorised-official-lookup            controllers.addressLookup.AuthorisedOfficialsAddressLookupController.initializeJourney(index: Index, mode: Mode = NormalMode)
```
Comment (abhijeetardale) [inline conf/address.routes:9]: please keep the consistent alignments

File: conf/officialsAndNominees.routes [hunk 2]
```diff
@@ -5,36 +5,50 @@ GET        /:index/authorised-official-name
 POST       /:index/authorised-official-name                                     controllers.authorisedOfficials.AuthorisedOfficialsNameController.onSubmit(mode: Mode = NormalMode, index: Index)
 GET        /:index/authorised-official-name/change                              controllers.authorisedOfficials.AuthorisedOfficialsNameController.onPageLoad(mode: Mode = CheckMode, index: Index)
 POST       /:index/authorised-official-name/change                              controllers.authorisedOfficials.AuthorisedOfficialsNameController.onSubmit(mode: Mode = CheckMode, index: Index)
+GET        /:index/authorised-official-name/playback-change                              controllers.authorisedOfficials.AuthorisedOfficialsNameController.onPageLoad(mode: Mode = PlaybackMode, index: Index)
```
Comment (abhijeetardale) [inline conf/officialsAndNominees.routes:8]: please keep the consistent alignments , for all the changes in file

**Review by abhijeetardale — CHANGES_REQUESTED** [review]
PR build failed for unit tests.
Please fix comilation warings "match may not be exhaustive."

File: conf/officialsAndNominees.routes (hunk 2 above)
Comment (RafSobol) [reply conf/officialsAndNominees.routes]: all fixed now

**Comment by RafSobol** [comment]
unit tests failed because of the change in frontendconfig (didn't realise there's even tests for strings in there!)

**Review by abhijeetardale — APPROVED** [review]

**Pull request merged**